### DISCUSSION TITLE
Is the stable / classic mod score debuff a permanent addition?

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#27166] [Q&A]
I am wondering if the current 0.96x score multiplier applied to scores set on stable and using the classic mod is going to be a permanent change?

The main issue I see with it is an arbitrary devaluation of many leaderboard scores. It does make sense for scores set without slider accuracy to receive a debuff, provided that they actually lost accuracy on sliders. If you overtake someone on the leaderboard who has the classic mod applied (and you don't), chances are they did just as well if not better than you. This ruins the competition in the leaderboards. If the leaderboards are ever merged completely, I don't think they will ever feel the same - there's something I find special about how well preserved the history of the leaderboards are.

The argument against this might be that if you want that then just use stable - which is the second issue I see; it makes a bigger divide between the two clients and makes user acceptance even harder. ~~If the score debuff is a permanent change, then I would suggest the addition of a classic leaderboard option in lazer, which displays the stable leaderboard and scores set in lazer which use the classic mod and any stable mod combinations (and that those scores also apply to the stable leaderboard).~~

**Comment by bdach** [comment]
related
https://github.com/ppy/osu/discussions/26899
https://github.com/ppy/osu/discussions/25859
https://github.com/ppy/osu/discussions/25903

**Comment by peppy** [comment]
> then I would suggest the addition of a classic leaderboard option in lazer, which displays the stable leaderboard and scores set in lazer which use the classic mod and any stable mod combinations (and that those scores also apply to the stable leaderboard).

you can turn on classic mod then go to mod leaderboards already, FWIW.

**Reply by ghost** [reply]
Most map leaderboards are the same as stable, so there's no need for a classic leaderboard option. It would take an enormous amount of time for the leaderboards of most maps to fill with lazer scores.

If lazer became the main client, first place ranks would be counted from the lazer leaderboard, which requires the leaderboard to be seen as at least equally competitive as stable's leaderboard. This is my basic line of reasoning in being concerned about the devaluation of leaderboard scores.

**Comment by Givikap120** [comment]
I've already with testing figured out that ideal multiplier for slider acc being disabled is 0.99
So it's most likely be the next value when we will get new multipliers

**Reply by bdach** [reply]
how did you "figure this out"

it's not "most likely" that it will be the "next value" if you come here and just say it will be without substantial arguments

**Reply by Givikap120** [reply]
Ran the CL replays without CL and compared the scores (ideally they should be the same)
Average was around 0.992, so 0.99 is close enough

**Reply by Rekunan** [reply]
Could you provide some examples of this? Just saying that the average is 0.992 isn't enough.

**Reply by Givikap120** [reply]
Here's some examples, trying to cover all edgecases:
Player, Map [Diff] - `converted score` -> `recalculated score` -> `score with slideracc` -> derived coef
maliszewski, Astral Empire [Break of Dawn] - 994254 -> 1000619 -> 999986 -> 0.999
Woey, Brain Power [Killer Machine] - 982787 -> 978955 -> 981606 ->1.003
Worst HR player, Exit This Earth Atmosphere Remix [Killer Machine] - 935718 -> 944147 -> 940795 -> 0.996
rng_, Feelin Sky [Ambivalence] - 834086 -> 827775-> 825579 ->0.997
tfge, HAWATARI NIOKU CENTI [CON] - 447798 -> 459666 -> 422116 -> 0.918
CutPaper, Impulse [Master] - 992405 -> 996298 -> 983901 -> 0.988
Gnahus, Packet Hero [Sotarks Yowasa] - 1054679 -> 1062445 -> 1042236 -> 0.981
CutPaper,  PEPSI MAN [MONSTRATA THE PEPSI ILLUMINATI] - 1017600 -> 1017600 -> 998776 -> 0.982
mrekk, Ray [Extreme] - 1071988 -> 1078846 -> 1060183 -> 0.983
maliszewski, Transhumanist [osu_ technology] - 762551 -> 762248 -> 708545 -> 0.930
Vaxei, Tsukinami [Nostalgia] - 1099943 -> 1108295 -> 1100474 -> 0.993
>

**Reply by Givikap120** [reply]
Take a note that i mostly took edgecases to get the lowest coef possible

**Reply by Rekunan** [reply]
0.99 probably looks good then yeah.

**Comment by ZeroAurora** [comment]
Not to create a new discussion I'm commenting here: In my experience, at least mania shouldn't have a 0.96x multiplier on classic plays. Mania is pretty an accuracy-focused mode with relatively small changes compared to stable. 0.96x multiplier means that a 98.5% acc play can override 99% acc classic ones. That's a lot.
